### PR TITLE
Add "Start with GoodData.UI" signpost

### PR DESCRIPTION
**Pull request by kandl**
- to simplify orientation in the documentation for newcomers
- few minor fixes for other doc pages

JIRA: RAIL-4057

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1045/ : SUCCESS in 1m 26s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/698/ : SUCCESS in 6s

File: docs/02_start__quickstart.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+### Integrate GoodData.UI into existing application
+If you already have JavaScript or TypeScript application, following articles should help you to integrate GoodData.UI into it:
+- [Integrate GoodData.UI into existing application for GoodData.CN](cloudnative_integration)
+- [Integrate GoodData.UI into existing application for GoodData Platform](platform_integration)
+- [Integrate GoodData.UI into Angular application](30_tips__use_angular_2.x.md)
+
+## Learn GoodData.UI
+
+### Export Catalog
```
Comment (no23reason) [inline docs/02_start__quickstart.md:23]: This should probably not be the first thing you learn. I would put it lower in the list.

File: docs/02_start__quickstart.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+### Execution model
+Understanding of the [execution model](understand_execution_model) is essential to work with GoodData.UI visualizations and creating your own.
+
+### Building and embedding data visualizations
+GoodData.UI offers multiple ways how to build or embed visualizations. [Choose the appropriate approach](choose_visualization_approach) that fulfills your needs and requirements. 
+
+### Building and embedding dashboards
+GoodData.UI offers multiple ways how to build or embed dashboards. You can also extend existing GoodData dashboards functionality.  [Choose the appropriate approach](choose_dashboard_approach) that fulfills your needs and requirements.
+
+### Learn by examples
```
Comment (no23reason) [inline docs/02_start__quickstart.md:35]: We could also add link to the Interactive examples

Comment (kandl) [reply docs/02_start__quickstart.md:35]: Added

File: docs/02_start__quickstart.md (hunk 1 above)
Comment (kandl) [reply docs/02_start__quickstart.md]: Good point. Updated.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1046/ : SUCCESS in 1m 21s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/699/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1053/ : SUCCESS in 1m 33s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/706/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1061/ : SUCCESS in 1m 24s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/713/ : SUCCESS in 6s

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result
 - No issue found. Congratulations!

More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-gate-PR515

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1064/ : SUCCESS in 1m 14s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/220/ : FAILURE in 2m 41s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sonarqube-scan-zuul-gate/115/ : SUCCESS in 38s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/310/ : SUCCESS in 21s

**Pull request merged**